javascript/new-src/bower.json

```diff
@@ -0,0 +1,45 @@
+{
+  "name": "OLA",
+  "version": "0.9.4",
+  "authors": [
+    "Dave Olsthoorn <dave.olsthoorn@gmail.com>"
+  ],
+  "description": "the OLA web-ui",
+  "keywords": [
+    "dmx",
+    "ola",
+    "open",
+    "lighting",
+    "architecture"
+  ],
+  "license": "MIT",
+  "private": true,
+  "ignore": [
+    "**/.*",
+    "node_modules",
+    "bower_components",
+    "test",
+    "tests"
+  ],
+  "dependencies": {
+    "angular": "~1.2.28",
+    "bootstrap": "~3.3.1",
+    "angular-route": "~1.2.x"
+  },
+  "exportsOverride": {
+    "angular": {
+      "js": ["*.min.js"]
+    },
+    "jquery": {
+      "js": "dist/*.min.js"
```

Comment:
Should this (and below) not be updated to point to where the file exists within OLA now?